CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added GitHub Action to generate MAPL3 Ford Docs
 - Added capability for HistoryCollectionGridComp to extract field names from expressions
 - Added ability for HistoryCollectionGridComp to extract multiple field names from expressions
+- Added procedures to get information about an ESMF_FieldBundle in History3G
+- Added module for keys to ESMF_Info metadata used in MAPL3G
```

Comment:
Well expressed comments. But ...  we are not tracking MAPL3 changes in the change log.

They would be misleading in the MAPL2 releases.  Instead, when we finally fold MAPL3 into develop we'll have to insert a brief description of MAPL3 and pointers to other documents.

```suggestion
```